import/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "import"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+futures = "0.3"
+object_store = { version = "0.3.0", features = ["aws", "rusoto_s3"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0.82"
+thiserror = "1.0.31"
+tokio = { version = "1.17" }
+tracing = { version = "0.1", features = ["max_level_trace", "release_max_level_debug"] }
+tracing-subscriber = "0.2.0"
```

I think we have been using the `observability_deps` (which imports tracing / tracing subscriber) elsewhere in the project

```suggestion
observability_deps = { path = "../observability_deps" }
```